src/interactive/Tooltip.jsx

```diff
@@ -94,15 +134,15 @@ class Tooltip extends React.PureComponent {
 		return (
 			<div
 				className={classNames.dropdown}
-				onMouseLeave={this.closeContent}
+				onMouseLeave={manualToggle ? undefined : this.closeContent}
```

Comment:
Nitpick - this and the following could be `manualToggle && this.closeContent`, though the ternary may be a little more explicit

Reply:
Agreed, though I believe being explicit is beneficial for future people reading code, rather than saving some time now 🙂